src/block_disagg/block_disagg_read.c

```diff
@@ -36,63 +37,64 @@ __wt_block_disagg_corrupt(
 }
 
 /*
- * __block_disagg_read --
- *     Read an addr/size pair referenced block into a buffer.
+ * __block_disagg_read_multiple --
+ *     Read a page referenced by a page id, checkpoint id, reconciliation id triple into multiple
```

Comment:
Please update the comment to mention that this is supposed to read the full image + all subsequent deltas.

Reply:
Done.